.dockerignore

```diff
@@ -0,0 +1,7 @@
+/__pycache__
+/.pytest_cache
+/.vagrant
+/.vscode
+/.venv
+.env
```

Comment:
Could write .env* to also ignore .env.test and .env.template

There are also a few more thing you could ignore, e.g .git